.github/workflows/docker.yml

```diff
@@ -5,4 +5,4 @@ on:
 
 jobs:
   docker:
-    uses: usdot-fhwa-stol/actions/.github/workflows/docker.yml@main
+    uses: usdot-fhwa-stol/actions/.github/workflows/docker.yml@main
```

Comment:
Is anything changed here. If not can we remove these changes.

Reply:
This change is for figuring out the merge conflict